gn3/computations/traits.py

```diff
@@ -0,0 +1,61 @@
+"""module contains all operating related to traits"""
+from gn3.computations.datasets import retrieve_trait_sample_data
+
+
+def compute_sum(rhs_val: int, lhs_val: int) -> int:
+    """initial function for trait module"""
+    return rhs_val + lhs_val
```

Comment:
*issue*: One line functions are a code smell. This just defines `+` no? :sweat_smile:

Reply:
This is just like an ice breaker for all my PRs not part of the codebase needs to be removed ASAP.